jbpm-test-coverage/src/test/java/org/jbpm/test/regression/task/HumanTaskTest.java

```diff
@@ -230,4 +238,31 @@ public void testInputTransformation() {
         assertProcessInstanceCompleted(pi.getId());
     }
 
+    @Test
+    public void testHumanTaskListener() {
+        KieSession ksession = createKSession(HUMAN_TASK_LISTENER);
+        AtomicBoolean triggered = new AtomicBoolean(false);
+        TaskLifeCycleEventListener listener = new DefaultTaskEventListener() {
+            @Override
+            public void afterTaskActivatedEvent(TaskEvent event) {
+                triggered.set(true);
+            }
+
+        };
+        RuntimeEngine engine = getRuntimeEngine();
+        TaskService taskService = engine.getTaskService();
+        ((EventService<TaskLifeCycleEventListener>) taskService).registerTaskEventListener(listener);
+        ProcessInstance pi = ksession.startProcess(HUMAN_TASK_LISTENER_ID);
+        long processInstanceId = pi.getId();
+
+        List<Long> idList = taskService.getTasksByProcessInstanceId(processInstanceId);
+        for (long taskId : idList) {
+            taskService.start(taskId, "john");
+            taskService.complete(taskId, "john", emptyMap());
```

Comment:
When completing first task, second task is added, so listener is invoked twice in this scenario

Reply:
changing this to watch how many times is triggered.